kubernetes/client.go

```diff
@@ -135,13 +144,24 @@ func NewClient() (*IstioClient, error) {
 // It returns an error on any problem.
 func NewClientFromConfig(config *rest.Config) (*IstioClient, error) {
 	client := IstioClient{}
+	log.Infof("Rest perf config QPS: %f Burst: %d", config.QPS, config.Burst)
```

Comment:
is there a reason QPS is logged with floating point precision %f but Burst is not? I've only seen these settings being set to integer values (e.g. 175, or 200, or whatever).

Reply:
QPS is represented as a float and Burst is an integer.
It sounded logical to use %f and %d here.